video/out/wayland_common.c

```diff
@@ -2062,7 +2062,11 @@ int vo_wayland_init(struct vo *vo)
      * before mpv does anything else. */
     wl_display_roundtrip(wl->display);
 
-    return true;
+    return 1;
```

Comment:
isn't the convention -1 err and 0 success?

Reply:
win32 and x11 both return 1 on success and 0 on failure.